specification/webpubsub/data-plane/WebPubSub/stable/2023-07-01/webpubsub.json

```
@@ -0,0 +1,1821 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Web PubSub Service REST API",
+    "version": "2023-07-01"
+  },
+  "paths": {
+    "/api/health": {
+      "head": {
+        "tags": [
+          "general"
+        ],
+        "summary": "Get service health status.",
+        "operationId": "HealthApi_GetServiceStatus",
+        "parameters": [
+          {
+            "in": "query",
+            "name": "api-version",
+            "description": "The version of the REST APIs.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The service is healthy"
+          },
+          "default": {
+            "description": "Error response"
+          }
+        },
+        "x-ms-examples": {
+          "HealthApi_GetServiceStatus": {
+            "$ref": "./examples/HealthApi_GetServiceStatus.json"
+          }
+        }
+      }
+    },
+    "/api/hubs/{hub}/:addToGroups": {
+      "post": {
+        "tags": [
+          "webpubsub"
+        ],
+        "summary": "Add filtered connections to multiple groups.",
+        "operationId": "WebPubSub_AddConnectionsToGroups",
+        "consumes": [
+          "application/json-patch+json",
+          "application/json",
+          "text/json",
+          "application/*+json"
+        ],
```

You should only allow `application/json`.  Supported all these other content types only adds complexity and opportunity for errors in both the service implementation and generated (or otherwise) client libraries. 
```suggestion
        "consumes": [
          "application/json"
        ],
```

---

updated